src/crypto/crypto_mbedtls-ec.c

```diff
@@ -294,27 +297,29 @@ int crypto_ec_point_add(
 	return ret ? -1 : 0;
 }
 
+static int tls_ctr_drbg_random(void *ctx, unsigned char *buf, size_t len)
```

Comment:
Why is it is required to feed random into mbedtls in this way?
We should have the required integration to get entropy alread.

Reply:
This is copying methodology in sockets_tls APIs in Zephyr. It is the same thing. TLS/DTLS  and legacy Mbed TLS APIs have frng/prng pointers they make use of.

Reply:
Ok, I can see that.

csrand may not be ctr_drbg. and sys_rand_get is not cryptographicaly secure.

I don't know if this is acceptable.

Reply:
This is a copy-paste of the same function that sockets_tls.c is using.